msteams-platform/apps-in-teams-meetings/teams-live-share-faq.md

```diff
@@ -40,7 +48,7 @@ Any data sent or stored through Fluid containers created by Live Share's hosted
 
 <summary><b>What meeting types does Live Share support?</b></summary>
 
-Currently, only scheduled meetings are supported and all participants must be on the meeting calendar. Meeting types such as, one-on-one calls, group calls, and meet now aren't supported.
+During Preview, only scheduled meetings are supported and all participants must be on the meeting calendar. Meeting types such as, one-on-one calls, group calls, and meet now aren't supported. Other meeting types will be supported in the future.
```

Comment:
Recommend removing the last line "Other meeting types will be supported in the future" as we don't state something that would be available in future.

Reply:
Updated as mentioned.